### ISSUE TITLE
Parameter with list returns validation error

### ISSUE THREAD

**Issue by dalecampbell** [ChristopheBougere/asl-validator#67]
Hello Again! When using parameter with a list in yaml, the validator returns an error stating the Parameter must be an object. The SFN deploys correctly without validation and produces the expected output. Using the serverless-step-function plugin with validation leads to the following error:

```
✕ State machine "passParameterList" definition is invalid:
  
  data.States['pass-execution-details'] should NOT have additional properties
  data.States['pass-execution-details'] should NOT have additional properties
  data.States['pass-execution-details'].Type should match pattern "^Parallel$"
  data.States['pass-execution-details'].Parameters should be object
  data.States['pass-execution-details'] should NOT have additional properties
  data.States['pass-execution-details'].Type should match pattern "^Task$"
  data.States['pass-execution-details'] should NOT have additional properties
  data.States['pass-execution-details'].Type should match pattern "^Map$"
  data.States['pass-execution-details'] should match exactly one schema in oneOf

```

**To Reproduce**

```yaml
service: test-parameter-with-list

provider:
  name: aws

plugins:
  - serverless-step-functions

stepFunctions:
  validate: true
  stateMachines:
    passParameterList: 
      name: mdw-source-system-director-${opt:stage}
      definition:
        StartAt: pass-execution-details
        States: 
          pass-execution-details:
            Type: Pass
            Parameters:
              - execution_id.$: $$.Execution.Id
                timestamp.$: $$.Execution.StartTime
            ResultPath: $.execution_details

```

I got the same error when testing with a Task step

**Expected behavior**
The following would be the output from this step. 

```
"execution_details": [
    {
      "execution_id": "arn:aws:states:us-east-1......",
      "timestamp": "2021-02-11T04:27:15.193Z"
    }
  ]
```

**Version:** [e.g. 1.9.4, can be found by running `asl-validator --version`]
```
└─┬ serverless-step-functions@2.29.0
  └── asl-validator@1.9.4 
```

**Comment by ChristopheBougere** [comment]
Hi @dalecampbell 
That should be fixed in #68, could you give a try to 1.9.5?

**Comment by dalecampbell** [comment]
I was able to test this today and it now works as expected. Thanks!
